spk/ffmpeg/Makefile

```diff
@@ -1,19 +1,21 @@
 SPK_NAME = ffmpeg
-SPK_VERS = 2.1.1
-SPK_REV = 1
+SPK_VERS = 3.0
+SPK_REV = 2
 SPK_ICON = src/ffmpeg.png
 DSM_UI_DIR = app
-CHANGELOG = update ffmpeg
+CHANGELOG = "1. Update ffmpeg to 3.0<br>2. Added 13 new libs/codecs<br>3. Support for yasm added"
```

Comment:
ffmpeg is at `3.0.2`. The third entry may be a bit much, you could add a whole lot more than just mentioning yasm if you go that route ;)